### PR TITLE
Test/smoke

### PR DESCRIPTION
**Pull request by VillSource**

### CODE REVIEW

File: src/main/java/nl/tudelft/jpacman/ui/NewPacManUI.java
Comment (VillSource) [inline src/main/java/nl/tudelft/jpacman/ui/NewPacManUI.java:1]: Add a comment

File: src/test/java/nl/tudelft/jpacman/NewUISmokeTest.java
Comment (VillSource) [inline src/test/java/nl/tudelft/jpacman/NewUISmokeTest.java:1]: Add a comment

File: src/test/java/nl/tudelft/jpacman/SinglePlayerPacmanFactoryTest.java [hunk 1]
```diff
@@ -50,6 +51,15 @@ public void testCreatePacmanFromFile() {
...
 
+    @Test
+    public void testCreatePacmanFromNotExistedFile() {
+        String map = "notExistedFile";
+
+        assertThrows(PacmanConfigurationException.class, ()->{
+            factory.createPacman("");
+        });
+    }
+
```
Comment (VillSource) [inline src/test/java/nl/tudelft/jpacman/SinglePlayerPacmanFactoryTest.java:65]: Add a comment

File: src/test/java/nl/tudelft/jpacman/NewUISmokeTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
+import java.awt.event.KeyEvent;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class NewUISmokeTest {
```
Comment (VillSource) [inline src/test/java/nl/tudelft/jpacman/NewUISmokeTest.java:35]: add comment

**Review by athenexfolk — APPROVED** [review]
OK

**Pull request merged**